strawberry/schema/schema_converter.py

```diff
@@ -295,11 +279,23 @@ def get_graphql_fields() -> Dict[str, GraphQLField]:
 
             return graphql_fields
 
+        is_type_of: Optional[Callable[[Any, GraphQLResolveInfo], bool]]
```

Comment:
`GraphQLResolveInfo` <- we should use StrawberryInfo, not sure we can do this easily here though, maybe we can do it after we do this issue: https://github.com/strawberry-graphql/strawberry/issues/1425